### ISSUE TITLE
Create Debian package

### ISSUE THREAD

**Issue by n8willis** [trufont/trufont#498]
It would be really nice to create installable Debian packages for Linux users. It looks as though there is a pretty-well tested utility for doing this automatically for a Python virtualenv: https://github.com/spotify/dh-virtualenv

**Comment by stultus** [comment]
@balasankarc

**Comment by aravindavk** [comment]
@copyninja

**Comment by copyninja** [comment]
just for note this has been requested here https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=806464 only some one need to package dependency

**Comment by n8willis** [comment]
Clarification: I was suggesting that dh-virtualenv be used to create an installable .deb package. Whether or not the Debian project picks it up for inclusion in the distribution is a policy decision on their part (and, likely, one that will take quite some time to complete). 

Whereas dh-virtualenv can create the package from the venv, dependencies included (AIUI).

**Comment by probonopd** [comment]
Maybe you want to give the [AppImage](https://appimage.org/) a try:
https://github.com/trufont/trufont/issues/624

The AppImage is roughly for Linux what the bundle is for the Mac.

**Comment by n8willis** [comment]
@probonopd You have already opened an issue of your own to promote AppImage; its success is not going to be furthered by hijacking this issue, about a different package distribution method, in addition.

**Comment by probonopd** [comment]
Just trying to be helpful by pointing to an alternative solution that is available as of today. I also linked to that issue.

**Comment by jalcine** [comment]
> Just trying to be helpful by pointing to an alternative solution that is available as of today. I also linked to that issue.

There's a difference between being helpful and being spammy.

**Comment by alexmyczko** [comment]
yeah deps are missing, otherwise i have something that doesn't work: http://phd-sid.ethz.ch/debian/trufont/ at version 0.5.0 (but builds), but todays daily checkout builds and works (didn't make a deb package though, itp is at wei)
